crates/edr_solidity/src/artifacts.rs

```diff
@@ -14,10 +14,10 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, thiserror::Error)]
 pub enum BuildInfoConfigError {
     /// JSON deserialization error
-    #[error(transparent)]
+    #[error("Failed to parse build info: {0}")]
```

Comment:
Out of curiosity: Can't a linter rule force you to add these messages?

Reply:
I think it's difficult, because sometimes you really just want to propagate the error message of the wrapped error.